src/pages/index.js

```diff
@@ -38,39 +89,83 @@ const profileEditModal = new PopupWithForm({
   },
 });
 
+profileEditModal.setEventListeners();
+
 const addCardModal = new PopupWithForm({
   popupSelector: "#card-edit-modal",
   handleFormSubmit: (data) => {
     console.trace(data);
+    addCardModal.setLoading(true);
+    api
+      .createCards(data)
+      .then((res) => {
+        cardSection.addItem(createCard(res));
+        Constants.cardEditForm.reset();
```

Comment:
`        Constants.cardEditForm.reset();
`

You have method `reset` inside `PopupWithForm`. Please, use it here


```js

        addCardModal.reset();

```